### ISSUE TITLE
Support multiple versions of MacOS and Xcode

### ISSUE THREAD

**Issue by greensky01** [cpisciotta/xcbeautify#23]
Update:

I have noticed xcbueatify cannot properly handle output from Xcode 11.2.1. So it may be a good idea to support different format of Xcode.

#################################

Could xcbeautify support different versions of OS and Xcode? If not yet, could these be implemented as new command line options?

Thanks!

**Comment by thii** [comment]
What is preventing it from supporting different versions of OS and Xcode? I suppose it runs on multiple OSes and can be built with different versions of Xcode.

**Comment by greensky01** [comment]
Hi @thii, I find it cannot give us as "beautiful" output on Xcode 11 as it can on Xcode 10. Do you have plan to work out a patch on this?

**Comment by thii** [comment]
Could you give me a reproducible example?

**Comment by greensky01** [comment]
It's fairly easy to reproduce.
1. Create a Cocoa application in Xcode (either 10 or 11 will do, and I name the project as FirstStep)
2. Navigate to the project folder, and do the following:
/path/to/Xcode10_2_1.app/Contents/Developer/usr/bin/xcodebuild | xcbeautify
You would see output similar to the following:

```
[FirstStep] Compiling AppDelegate.m
[FirstStep] Compiling main.m
[FirstStep] Linking FirstStep
[FirstStep] Compiling MainMenu.xib
[FirstStep] Processing Info.plist
[FirstStep] Generating FirstStep.app.dSYM
Signing FirstStep.app (in target: FirstStep)
[FirstStep] Touching FirstStep.app
Build Succeeded
```

3. Clean and build with Xcode 11:
/path/to/Xcode11_2_1.app/Contents/Developer/usr/bin/xcodebuild | xcbeautify
You would see:

```
Processing Info.plist
Signing FirstStep.app (in target 'FirstStep' from project 'FirstStep')
Build Succeeded
```

We lost many outputs.

**Comment by thii** [comment]
@greensky01 Could you try the version built from the `xcode11-outputs` branch to see if it works for you?

**Comment by greensky01** [comment]
@thii I've tried your code and can confirm the issue's been fixed. Thanks!

**Comment by thii** [comment]
Thanks. I merged it.

**Comment by thii** [comment]
I've released version 0.7.0 that includes the fixes.
